prow/gerrit/gerrit.go

```diff
@@ -294,14 +299,30 @@ func (c *Controller) ProcessChange(change gerrit.ChangeInfo) error {
 		if _, err := c.kc.CreateProwJob(pj); err != nil {
 			logger.WithError(err).Errorf("fail to create prowjob %v", pj)
 		} else {
-			triggered = append(triggered, spec.Name)
+			var b bytes.Buffer
+			template := c.ca.Config().Plank.JobURLTemplate
+			if template != nil {
+				if err := template.Execute(&b, &pj); err != nil {
+					logger.WithFields(pjutil.ProwJobFields(&pj)).Errorf("error executing URL template: %v", err)
+				}
+			}
+			triggeredJobs = append(triggeredJobs, triggeredJob{Name: spec.Name, URL: b.String()})
 		}
 	}
 
-	if len(triggered) > 0 {
+	if len(triggeredJobs) > 0 {
 		// comment back to gerrit
+		message := "Triggered presubmit:"
+		for _, job := range triggeredJobs {
+			if job.URL != "" {
+				message += fmt.Sprintf("\n  * Name: %s, URL: %s", job.Name, job.URL)
+			} else {
+				message += fmt.Sprintf("\n  * Name: %s, URL: unknown", job.Name)
```

Comment:
nit: "URL: unknown" doesn't seem particularly useful. It would probably be a little cleaner to omit it altogether.

/lgtm
/hold

Reply:
sure :-)